tests/testthat/test-renderer4-prostateLasso.R

```diff
@@ -30,13 +30,11 @@ viz.no.time <- list(
       xmin=arclength.click-rect.width,
       xmax=arclength.click+rect.width,
       id=paste0("arclength", round(arclength.click, 1)*10),
-      clickSelects.variable="arclength",
-      clickSelects.value=arclength.click,
-      showSelected.variable="arclength",
-      showSelected.value=arclength.show,
       key=ifelse(
         arclength.click==arclength.show, 1,
         paste(arclength.click, arclength.show))),
+      clickSelects=c(arclength="arclength.click"),
+      showSelected=c(arclength="arclength.show"),
       alpha=0.5,
       data=addY(tallrect.all, "error")),
   res=ggplot()+
```

Comment:
@tdhock Here when I converted to the newer syntax, I noticed the data `addY(tallrect.all, "error")`, has no `arclength` column and hence we [encounter this error in the tests](https://travis-ci.org/tdhock/animint2/jobs/264712918#L2955-L2958). I guess just adding the `arclength` column separately should solve this, but I think I should add a separate test to ensure this throws an error in our code. Any comments on this?